pymongo/client_bulk_shared.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2024-present MongoDB, Inc.
```

Comment:
Can you rename this file to `_client_bulk_shared.py` to make it more clearly private?